docker/start-containter-with

```diff
@@ -12,5 +10,5 @@ docker run --rm -ti \
   --mount type=bind,source="$data",target=/pacta-data \
   --mount type=bind,source="$report",target=/create_interactive_report \
   --mount type=bind,source="$stresstest",target=/StressTestingModelDev \
-  2diirunner:1.3.0 "$1"
+  maurolepore/2diirunner-with-packages $1
```

Comment:
This is the key. Now we run the image that already includes all required R packages.